.travis.yml

```diff
@@ -40,7 +40,6 @@ before_script:
 
 script:
   - 'if [ "$KARMA" = "true" ]; then bundle exec rake karma:run; else echo "Skipping karma run"; fi'
-  #- "KNAPSACK_GENERATE_REPORT=true bundle exec rspec spec"
```

Comment:
Can we document somewhere why and how we run this?

Reply:
Yep, we should add this somewhere in the wiki

Reply:
Done. You'll find it under the testing section of the wiki. https://github.com/openfoodfoundation/openfoodnetwork/wiki/Parallelized-test-suite